config/templates/package-scripts/prerm.erb

```diff
@@ -51,6 +51,8 @@ when "aix" -%>
   done
 <% when "freebsd" -%>
 # this is a noop on <%= platform_family %>
+<% when "mac_os_x" -%>
```

Comment:
Consider combining with above case, e.g. `when "freebsd", "mac_os_x"`, for brevity?